src/app/lib/utilities/activateExperiment/index.js

```diff
@@ -0,0 +1,14 @@
+import onClient from '#lib/utilities/onClient';
+
+const activateExperiment = async (optimizely, experimentName, variation) => {
+  if (onClient()) {
+    await optimizely.onReady().then(({ success }) => {
```

Comment:
Any way this could avoid using promises? I'm wondering about adding a coding standard around this 🤔